intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -116,14 +116,14 @@ LLM running script args explanations:
 | -p / --prompt     | prompt to start generation with (default: empty)                        |
 | -n / --n_predict  | number of tokens to predict (default: -1, -1 = infinity)                |
 | -t / --threads    | number of threads to use during computation (default: 56)               |
-| -b / --batch_size | batch size for prompt processing (default: 512)                         |
+| -b / --batch_size_truncate | batch size for prompt processing (default: 512)                         |
```

Comment:
is it truncate for batch size or prompt length?

Reply:
truncate for prompt length